bloom-housing/bloom
# feat: adds updating open listing modal

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2216)


## Description
Adds a new modal when saving during editing an open listing 

## How Can This Be Tested/Reviewed?
Create a new listing that is published or edit an existing active listing 
make modifications (or don't make modifications)
click the save & exit button and the new modal should pop up

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
